applications/trilinos_application/custom_utilities/trilinos_mvqn_recursive_convergence_accelerator.hpp

```diff
@@ -0,0 +1,769 @@
+//    |  /           |
```

Comment:
...yes it does :-(. We will need to cleanup this, although at a later stage

Reply:
Yes we do...